### ISSUE TITLE
Scrum Update 1

### ISSUE THREAD

**Issue by mandywan** [kenhyj/Traceify#6]
What did you work on this past iteration (2 weeks)?

What were any major issues/challenges you ran into?

What do you plan to work on for this coming iteration (2 weeks)?

**Comment by kenhyj** [comment]
CSID: e5c0b. 
For this past iteration, I with our team compiled our prototype sketches and Tables schema that could be used to achieve some of our implementations.
Major issues I had was understanding how to get Google Map work with our mock data, since our team do not have any experience using Google Map's API
For this coming iteration, I plan on making our front-end user-end for our Home, Thank you, Symptom Checker, Have I been exposed?,  Financial Helper. The to do list will be adjusted accordingly to our Kanban board listed in our team Trello.

**Comment by kessris** [comment]
1) Our group came up with the project idea, its prototype,

**Comment by kessris** [comment]
CSID: k0b0b

1) Engaged in a group meeting to decide on our project idea. We constantly communicated to sort out the project details, database table schemas, set-up the Trello board for tracking tasks, set up the Github repo, and make the prototypes.

2) We still haven't found a solution on how to make the Admin panel accessible only to the admins - we need to discuss more on that and decide on a way soon.

3) Break up and list out all the tasks on the Trello board and assign them to each of the members. Set-up the general environment (React, Redux, install DB drivers, etc.). Start creating few re-usable components (i.e., header, buttons, etc.)

**Comment by ChaoyuXiang** [comment]
CSID: l0y2b
1. For the first sprint, we discussed about the idea of our projects and came up basic data scheme and prototype.
2. Major issue for me was I haven't learned much about database stuff and I had some difficulties understanding inner structure of database and how it works/ 
3. For the next sprint, we're gonna adapt react-redux into our project and start to implement our first home page.

**Comment by mandywan** [comment]
csid: d2e2b

This past iteration, we worked on the project idea together. I signed up for a Google Maps API key. We also came up with prototypes and the requirements.
We still have not come to a collective agreement on how to secure the Admin Panel, and I still need to figure out how to connect Google Maps to our application.
For the next sprint, we will implement the UI for our prototypes using React and Redux.
